Problema3.txt

```diff
@@ -0,0 +1,29 @@
+public bool ThereAreDuplicates(int[] a)
+        {
+            int gaussSum = 0, actualSum = 0;
+            long multiply = 1, factorial = 1;
+            int size = a.Length;
+            for (int i = 0; i < size; i++)
+            {
+                actualSum += a[i];
+                multiply *= a[i];
+                factorial *= (i + 1);
+            }
+            gaussSum = size * (size + 1) / 2;
+            if (gaussSum != actualSum)
+
+                return true;
+            else
+                if (multiply != factorial)
+                    return true;
+            return false;
+        }
+        
+          public void Arrays_WithDuplicates_ShouldFail()
+        {
+            int[] a = new int[] {2,2,6,1,3,3,5};
+            Methods method = new Methods();
```

Comment:
Ah da, am pus doar efectiv metoda care rezolva problema, Methods e o clasa
in care exista metoda ThereAreDuplicates care primeste un array. iar
Arrays_WithDuplicates_ShouldFail e din interiorul unit testing-ului

On Mon, Mar 30, 2015 at 12:52 PM, Calinoiu Alexandru Nicolae <
notifications@github.com> wrote:

> In Problema3.txt
> <https://github.com/Agilefreaks/Test/pull/9#discussion_r27377597>:
>
> > +                factorial *= (i + 1);
> > +            }
> > +            gaussSum = size * (size + 1) / 2;
> > +            if (gaussSum != actualSum)
> > +
> > +                return true;
> > +            else
> > +                if (multiply != factorial)
> > +                    return true;
> > +            return false;
> > +        }
> > +
> > +          public void Arrays_WithDuplicates_ShouldFail()
> > +        {
> > +            int[] a = new int[] {2,2,6,1,3,3,5};
> > +            Methods method = new Methods();
>
> Cred ca undeva e un Methods class aici nu se vede.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Agilefreaks/Test/pull/9/files#r27377597>.
>